org.elixir_lang.psi.scope.Module.aliasedName(Module.kt:52) AssertionError

# Version

7.5.0


# Exception


## Message

Assertion failed


## Stacktrace

```
java.lang.AssertionError: Assertion failed
	at org.elixir_lang.psi.scope.Module.aliasedName(Module.kt:52)
	at org.elixir_lang.psi.scope.Module.aliasedName(Module.kt:43)
	at org.elixir_lang.psi.scope.Module.executeOnAliasCallArgument(Module.kt:147)
	at org.elixir_lang.psi.scope.Module.executeOnAliasCallArgument(Module.kt:85)
	at org.elixir_lang.psi.scope.Module.executeOnAliasCall(Module.kt:66)
	at org.elixir_lang.psi.scope.Module.execute(Module.kt:34)
	at org.elixir_lang.psi.scope.module.MultiResolve.execute(MultiResolve.kt:25)
	at org.elixir_lang.psi.impl.ProcessDeclarationsImpl.processDeclarationsInPreviousSibling(ProcessDeclarationsImpl.kt:290)
	at org.elixir_lang.psi.impl.ProcessDeclarationsImpl.processDeclarations(ProcessDeclarationsImpl.kt:145)
	at org.elixir_lang.psi.impl.ElixirPsiImplUtil.processDeclarations(ElixirPsiImplUtil.java:592)
	at org.elixir_lang.psi.impl.ElixirStabBodyImpl.processDeclarations(ElixirStabBodyImpl.java:63)
	at com.intellij.psi.util.PsiTreeUtil.treeWalkUp(PsiTreeUtil.java:1120)
	at org.elixir_lang.psi.scope.module.MultiResolve$Companion.resolveResults(MultiResolve.kt:105)
	at org.elixir_lang.psi.scope.module.MultiResolve$Companion.resolveResults(MultiResolve.kt:80)
	at org.elixir_lang.reference.resolver.Module.resolve(Module.kt:18)
	at org.elixir_lang.reference.resolver.Module.resolve(Module.kt:10)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolve(ResolveCache.java:152)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:182)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:174)
	at org.elixir_lang.reference.Module.multiResolve(Module.kt:29)
	at org.elixir_lang.psi.impl.QualifiableAliasImplKt.fullyResolve(QualifiableAliasImpl.kt:46)
	at org.elixir_lang.psi.impl.QualifiableAliasImplKt.toModular(QualifiableAliasImpl.kt:150)
	at org.elixir_lang.psi.impl.QualifiableAliasImplKt.maybeModularNameToModular(QualifiableAliasImpl.kt:138)
	at org.elixir_lang.psi.impl.PsiElementImplKt.maybeModularNameToModular(PsiElementImpl.kt:134)
	at org.elixir_lang.psi.impl.call.qualification.QualifiedImplKt.qualifiedToModular(QualifiedImpl.kt:19)
	at org.elixir_lang.reference.resolver.Callable.resolveElement(Callable.kt:61)
	at org.elixir_lang.reference.resolver.Callable.resolve(Callable.kt:18)
	at org.elixir_lang.reference.resolver.Callable.resolve(Callable.kt:13)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolve(ResolveCache.java:152)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:182)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:174)
	at org.elixir_lang.reference.ResolveWithCachingComputable.compute(Callable.kt:596)
	at org.elixir_lang.reference.ResolveWithCachingComputable.compute(Callable.kt:588)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:913)
	at org.elixir_lang.reference.CallableKt.resolveWithCaching(Callable.kt:586)
	at org.elixir_lang.reference.CallableKt.access$resolveWithCaching(Callable.kt:1)
	at org.elixir_lang.reference.Callable.multiResolve(Callable.kt:109)
	at org.elixir_lang.annotator.Callable$annotate$1.visitPlainCall(Callable.kt:96)
	at org.elixir_lang.annotator.Callable$annotate$1.visitNonModuleAttributeCall(Callable.kt:85)
	at org.elixir_lang.annotator.Callable$annotate$1.visitCall(Callable.kt:46)
	at org.elixir_lang.annotator.Callable$annotate$1.visitElement(Callable.kt:77)
	at com.intellij.psi.impl.PsiElementBase.accept(PsiElementBase.java:274)
	at org.elixir_lang.psi.impl.ElixirMatchedQualifiedParenthesesCallImpl.accept(ElixirMatchedQualifiedParenthesesCallImpl.java:38)
	at org.elixir_lang.annotator.Callable.annotate(Callable.kt:38)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.a(DefaultHighlightVisitor.java:139)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.visit(DefaultHighlightVisitor.java:102)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:371)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:303)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:330)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:333)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.analyze(DefaultHighlightVisitor.java:86)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:333)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.a(GeneralHighlightingPass.java:300)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectInformationWithProgress(GeneralHighlightingPass.java:239)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:83)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:70)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.a(PassExecutorService.java:437)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1134)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.d(PassExecutorService.java:430)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:580)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:525)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:85)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.b(PassExecutorService.java:429)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.c(PassExecutorService.java:405)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:143)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:218)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:403)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:170)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)

```

--- a/src/org/elixir_lang/psi/scope/Module.kt
+++ b/src/org/elixir_lang/psi/scope/Module.kt
@@ -49,8 +49,6 @@ abstract class Module : PsiScopeProcessor {
         val operatorIndex = org.elixir_lang.psi.operation.Normalized.operatorIndex(children)
         val unqualified = org.elixir_lang.psi.operation.infix.Normalized.rightOperand(children, operatorIndex)
 
-        assert(children.size == 3)
-
         return (unqualified as? PsiNamedElement)?.name
     }
 

